packages/embeds/codesandboxes/embed-react/CSS Variables API/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "CSS Variables API",
+  "packageManager": "yarn@3.4.1",
+  "scripts": {
+    "type-check": "tsc --pretty --noEmit"
+  },
+  "dependencies": {
+    "@calcom/embed-react": "workspace:^"
```

Comment:
We need workspace: here so that we can test with latest changes from @calcom/embed react that aren't yet published.

But for the synced codesandbox to work it has to be a an npm repository depedency, something like '1.0.1'